api/src/main/scala/dx/api/DxApi.scala

```diff
@@ -1089,12 +1092,15 @@ case class DxApi(version: String = "1.0.0", dxEnv: DXEnvironment = DxApi.default
     * @param dxfile the dx file to download
     * @param overwrite whether to overwrite an existing file - if false, an
     *                  exception is thrown if the target path already exists
+    * @param retryLimit max number of retries
+    * @param cliRunner max number of retries
```

Comment:
It looks like a copy/paste error (?)

Reply:
fixed in [e5e0606](https://github.com/dnanexus/dxScala/pull/112/commits/e5e06067986c57573d0d27d95701f523761a645d)